tests/rt.conf

```diff
@@ -123,6 +123,9 @@ RUN | control_iovr5                                     |
 RUN | control_p8                                        | - noaacloud                          | baseline |
 RUN | control_p8.v2.sfc                                 | - noaacloud                          | baseline |
 RUN | control_p8_ugwpv1                                 | - noaacloud                          | baseline |
+RUN | control_p8_ugwpv1_tempo                           | - noaacloud                          | baseline |
```

Comment:
@AndersJensen-NOAA These new RTs are failing because the new suite "FV3_GFS_v17_p8_ugwpv1_tempo" is not being compiled above in this file. I'll fix this.